### PR TITLE
Allows control "production" env passed to compass

### PR DESCRIPTION
**Pull request by lukaszlenart**
Right now it isn't possible to control the `production` flag which is always passed to compass and because of that any configuration that uses the flag in `config.rb` is useless.

### CODE REVIEW

File: src/test/scala/com/openstudy/sbt/SassCompilationSpec.scala [hunk 1]
```diff
@@ -77,6 +79,38 @@ class SassCompilationSpec extends FunSpec with MockitoSugar {
           mockBaseDirectory
         )
       }
+
+      it("with production off") {
```
Comment (Shadowfiend) [inline src/test/scala/com/openstudy/sbt/SassCompilationSpec.scala]: Lol. I have no idea why I went with scalatest in this project haha.

**Review by Shadowfiend — APPROVED** [review]
LGTM. Have y'all given this a test whirl with a real project @lukaszlenart ?

**Comment by lukaszlenart** [comment]
Yeah... but it doesn't work, I mean I cannot configure sbt to properly pass the flag to the plugin. When I'm testing with sbt shell I see:

```
> resources:compile-sass::productionSassCompile
[info] true
```

but once I run the task:

```
> resources:compile-sass
[info] Compiling SASS files using: compass compile
[info] Done.
```

there is no `-e production` flag :(

**Comment by lukaszlenart** [comment]
Looks like this is an issue how the settings are defined in the plugin ...

**Comment by lukaszlenart** [comment]
@Shadowfiend fixed, should I also add the proper scope to the rest of settings? Without the scope they won't work (as far I understand).

**Comment by Shadowfiend** [comment]
I've used the other settings before though… 🤔 I wonder if this is a change in sbt (this project is pretty old).

I'd say if you can confirm they don't work without scope changes, go for it. If not, let's not worry about it. I think we can also merge this without any of those tweaks, unless you're aching to look at the other settings :)

**Comment by lukaszlenart** [comment]
I mean, the AWS related settings won't work - we don't use them in our project. The other setting are properly scoped.

**Comment by Shadowfiend** [comment]
Right, sorry, I mean in past projects I've used those---and so has @farmdawgnation I believe---without trouble. That's why I'm wondering if this is an issue with newer sbt versions that didn't exist in older ones.

**Comment by farmdawgnation** [comment]
SBT changed a lot of things in 0.13.15,16 and up. May be worth seeing if there's different behavior in an older sbt version.

**Comment by lukaszlenart** [comment]
I have digged a bit around scopes in sbt and I can use `Global` to fix the problem without changing the plugin, i.e. `productionSassCompile in Global := true` - this will work as expected. What I do not understand why `productionSassCompile := true` doesn't work :(

**Comment by lukaszlenart** [comment]
Hm.... `productionSassCompile in ResourceCompile` also works 🤔

**Comment by lukaszlenart** [comment]
Triple tested in my project and works as expected, can be merged :)

**Comment by Shadowfiend** [comment]
Okay haha. Let's do it!

**Pull request merged**